clippy_lints/src/methods/mod.rs

```diff
@@ -1633,6 +1634,35 @@ declare_clippy_lint! {
     "replace `.iter().count()` with `.len()`"
 }
 
+declare_clippy_lint! {
+    /// **What it does:** Checks for calls to `splitn` and related functions with
```

Comment:
Please insert a link to the ``splitn`` method here.

```suggestion
    /// **What it does:** Checks for calls to [`splitn`](https://doc.rust-lang.org/std/primitive.str.html#method.splitn) and related functions with
```